lib/logdna.rb

```diff
@@ -142,18 +111,5 @@ def datetime_format(*arg)
       puts "datetime_format not supported in LogDNA logger"
       return false
     end
-
-
-    def close
```

Comment:
Can you explain why you removed these?

Reply:
Could you explain that to me? What do you mean it doesn't do anything? `close` is the override for the parent class (`Logger::close`). `at_exit` is called on program exit. Are you saying `@@client.exitout()` is a no-op?